tests/Gedmo/Mapping/SluggableMappingTest.php

```diff
@@ -49,10 +48,6 @@ protected function setUp(): void
             'Gedmo\Tests\Mapping\Fixture\Yaml'
         );
         $reader = new AnnotationReader();
-        AnnotationRegistry::registerAutoloadNamespace(
-            'Gedmo\\Mapping\\Annotation',
-            dirname(VENDOR_PATH).'/src'
```

Comment:
I think this is the only use for this constant. IMO, we can also safely remove its definition.

Reply:
Done